app/views/releases/index.html.erb

```diff
@@ -19,6 +19,7 @@
         <thead>
           <tr>
             <th></th>
+            <th>Status</th>
```

Comment:
unclear that this is github status ...should say github status or have an octopus icon or a hover

Reply:
I was actually considering removing the `th`, since it takes up more space than the icons themselves. How about a hover text saying e.g. "Github status: failed"?